public/app/features/explore/Logs.tsx

```diff
@@ -353,12 +353,12 @@ export class UnthemedLogs extends PureComponent<Props, State> {
           </div>
           <LogsNavigation
             logsSortOrder={logsSortOrder}
-            visibleRange={visibleRange}
+            visibleRange={visibleRange || absoluteRange}
             absoluteRange={absoluteRange}
             timeZone={timeZone}
             onChangeTime={onChangeTime}
             loading={loading}
-            queries={queries}
+            queries={logsQueries || []}
```

Comment:
```suggestion
            queries={logsQueries}
```

it seems to me that `logsQueries` can never be undefined based on https://github.com/grafana/grafana/blob/ivana/navigation-refresh/public/app/features/explore/Logs.tsx#L50 (change untested)

Reply:
I decided to follow the pattern of other logs params and made it optional params. And therefore here, I have added `logsQueries ?? []`
https://github.com/grafana/grafana/pull/33971/commits/2fd0e8e9aabee04595dd865a50c9f157c3b90187